einstein/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Define the version of the template module.
-__version__ = "1.1.1"
+__version__ = "1.0.1"
```

Comment:
please make sure you always pull the latest version of the repo, I update quite regularly